tracee/tracee.go

```diff
@@ -367,15 +370,26 @@ func (t *Tracee) processEvent(ctx *context, args []interface{}) error {
 	if t.config.CaptureFilesExec && (eventName == "security_bprm_check") {
 		var err error
 
-		destinationDirPath := filepath.Join(t.config.OutputPath, strconv.Itoa(int(ctx.Mnt_id)))
-		if err := os.MkdirAll(destinationDirPath, 0755); err != nil {
-			return err
-		}
 		sourceFilePath, ok := args[0].(string)
 		if !ok {
 			return fmt.Errorf("error parsing security_bprm_check args")
 		}
+		sourceFile, err := os.Stat(sourceFilePath)
+		if err != nil {
+			return err
+		}
+		lastMtime, ok := t.capturedFiles[sourceFilePath]
+		if ok && lastMtime == sourceFile.ModTime() {
```

Comment:
Don't use modification time, as this can be altered by the user.
Use ctime (change time), which is altered by the kernel, also on metadata change